src/analyses/base.ml

```diff
@@ -2093,7 +2094,6 @@ struct
     in
     let st = match desc.special args, f.vname with
     | Memset { dest; ch; count; }, _ ->
-      (* TODO: check count *)
```

Comment:
Might make sense to keep the `check_count` TODO, I am not sure the `set` here does the right thing, even if the size is within bounds.

Reply:
Should be fixed in https://github.com/goblint/analyzer/pull/1197/commits/89abb1a9ade5352f19fc2c042c4ef2b387c5fa38